### PR TITLE
Feat/support paseo #1881

### LINKED ISSUE #1881: Paseo Testnet Support
We are adding support for the new [Paseo Testnet](https://github.com/paseo-network/)!

**Pair strongly suggested**

## Notes / Separate Issues
- Do NOT remove Rococo for now
- e2e Tests should still run against Live Rococo.
- Testnet token is `XRQCY` on all public testnets. (`UNIT` on Local Testnet)
- [Paseo Spec](https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/polkadot/node/service/chain-specs/paseo.json)
    - `protocolId` = `pas`

## Checklist
- [x] Paseo chain spec
- [x] Binary builds in the Release
- [ ] Docker builds
- [x] Rename feature `frequency-rococo-testnet` to `frequency-testnet`
- [ ] Update Readme
- [ ] Generate new Spec File (Using CI) and add it
- [x] Rename "Rococo Local Testnet" -> "Local Testnet"
- [ ] Update `.env.sample`
- [ ] Update Make commands
- [x] Bootnodes
    - [ ] Coming...

### PR DESCRIPTION
**Pull request by shannonwells**
# Goal
The goal of this PR is to add support for running a Paseo testnet.

## TODOs
- [x] Add bootnodes to spec
- [x] Generate raw spec
- [x] Parachain slot
- [ ] Generate genesis and wasm for onboarding

Closes #1881 

# Discussion

# Checklist
- [x] Chain spec updated
- [ ] Tests added
- [ ] Benchmarks added
- [ ] Weights updated

### CODE REVIEW

File: docker/parachain-node-rococo.overview.md
Comment (wilwade) [inline docker/parachain-node-rococo.overview.md:1]: Replaced with `parachain-node-testnet`

File: node/service/src/chain_spec/frequency_paseo.rs [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+	pub const FRQ_COUNCIL6: &str =
+		"0xe87a126794cb727b5a7760922f81fbf0f80fd64b7e86e6ae4fee0be4289c7512";
+	pub const FRQ_COUNCIL7: &str =
+		"0x14a6bff08e9637457a165779765417feca01a2119dec98ec134f8ae470111318";
+	pub const FRQ_COUNCIL8: &str =
+		"0x140c17ced6e4fba8b62a6935052cfb7c5a8ad8ecc43dee1f4fc7c30c1ca3cb14";
+	pub const FRQ_COUNCIL9: &str =
+		"0xfc61655783e14b361d2b9601c657c3c5361a2cf32aa1a448fc83b1a356808a1a";
+	pub const FRQ_COUNCIL10: &str =
+		"0x1caccabbe8095a35292782cffc29145030264f9706753923a5c237e8f5aceb1a";
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency_paseo.rs:68]: We need to remove the 10th so it is an odd number.
```suggestion
```

**Review by wilwade — APPROVED** [review]
✅ when CI is ✅

**Review by saraswatpuneet — APPROVED** [review]

**Review by JoeCap08055 — APPROVED** [review]
Just so I understand, this PR:
* Changes all build feature flags (ie, `Cargo.toml` stuff) to use 'testnet' instead of 'rococo'
* Adds appropriate specs, etc, for Paseo
* For build pipeline flags/args that pertain to deployment (as opposed to features), duplicate 'testnet' references for 'paseo' and 'rococo'

Great work! I appreciate all of the tedium that must have been involved in chasing down all of the spots to change.

**Pull request merged**